components/Registration/Mitra/index.vue

```diff
@@ -333,6 +341,21 @@ export default {
       }
     }
   },
+  mounted () {
+    const urlParams = window.location.hash
+    if (urlParams) {
+      const queryparams = urlParams.split('#')[1]
+      const params = queryparams.split('&')
+      let pair = null
+      const data = []
+      params.forEach(function (d) {
+        pair = d.split('=')
+        data.push({ key: pair[0], value: pair[1] })
+      })
```

Comment:
hello @doohanas , can't we achieve the same result using `this.$route` ? I believe it returns an object that contains the information of the URL so we don't have to split the URL strings 🤔

Reply:
Hello kak @Ibwedagama, yes, you're right I can achieve that with `this.$route` but I have the same issue after the user login and redirect to another page and the state is (see picture below)
![image](https://user-images.githubusercontent.com/79241754/161895514-f47089ad-75e6-4d62-8850-54d721487e1e.png).
And I have the same issue with [this](https://github.com/nuxt-community/auth-module/discussions/1502). 
So, the solution, for now, I split the URL strings and get the `access_token` and pass in `fetchUser` method

Reply:
Hello kak @Ibwedagama, yes, you're right I can achieve that with `this.$route` but I have the same issue after the user login and redirect to another page and the state is (see picture below)
![image](https://user-images.githubusercontent.com/79241754/161895514-f47089ad-75e6-4d62-8850-54d721487e1e.png).
And I have the same issue with [this](https://github.com/nuxt-community/auth-module/discussions/1502). 
So, the solution, for now, I split the URL strings and get the `access_token` and pass in `fetchUser` method

Reply:
I see, okay just stick with your solution bro 👍🏼 👍🏼